feat: add env tag doc (#511)
logto-io/docs: docs/docs/recipes/environment-tags/README.mdx

import CloudOnlyNotification from '../../fragments/_cloud_only_notification.mdx';

# 🏷️ Environment Tags

<CloudOnlyNotification feature="environment tag" />

Logto just enhanced the current environment tag features and introduced a new type of tenant: the Development tenant.

## Type of tenants

There are two types of tenants in Logto Cloud.

### Development

The dev environment is mainly used for testing and should not be used in the production environment. Similar to the Pro plan, it's free forever with some limitations and no subscription required.

### Production

Production is where end-users access the live software and might need a paid subscription. To access a production tenant, you can subscribe to the free, hobby, or pro plan.

:::note
Please be aware that you can only specify your tenant types during creation; it is not possible to modify the tenant type once it has been created.
:::

With the new type of tenant differentiation, you can better manage your projects across different environments for efficiency and, at the same time, enjoy the full value of Logto.

## Legacy migration

- Users on the free plan with environment tags "dev" or "prod" will remain unaffected.

> - Free plan users with environment tags set as "staging" will experience the sunset of the staging environment. This tenant type will change from “staging” to “production”, with no impact on the current setup.

Reviewer: ```suggestion
- "Staging" environment tag will be sunset, so free plan tenants with environment tags set as "staging" will be automatically changed to “production”.
```
